src/macros.rs

```diff
@@ -93,20 +93,38 @@ macro_rules! prelude {
 /// Implement `Clone` and `Copy` for a struct, as well as `Debug`, `Eq`, `Hash`, and
 /// `PartialEq` if the `extra_traits` feature is enabled.
 ///
+/// By default, it will mark a struct as `non_exhaustive`. To opt out, use the attribute
+/// `#[@not_non_exhaustive]` as the first attribute of the struct.
+///
 /// Use [`s_no_extra_traits`] for structs where the `extra_traits` feature does not
 /// make sense, and for unions.
 macro_rules! s {
     ($(
+        $(#[@$not_non_exhaustive:ident])*
```

Comment:
This should just be able to match a literal, right?

```suggestion
        $(#[not_non_exhaustive])?
```

Reply:
Nope, it fails with `"attempted to repeat an expression containing no syntax variables"`